server/dash/dashboards/neighborhood.py

```diff
@@ -39,27 +42,80 @@ def populate_options():
         )
     return values
 
+print(populate_options())
+
+layout = html.Div([
+    html.H1(title),
+    dcc.Dropdown(
+        id='council_list',
+        clearable=False,
+        value="Arleta",
+        placeholder="Select a neighborhood",
+        options=populate_options()
+    ),
+    dcc.Dropdown(
```

Comment:
Mentioned this in person, but it would be nice to make this dropdown a component of the first graph since it only applies to that graph.

Reply:
I made a new pull request with radio options now found below the graph.